cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+'use strict';
+var meow = require('meow');
+var logSymbols = require('log-symbols');
+var np = require('./');
+
+var cli = meow(`
+	Usage
+	  $ np [patch | minor | major | <version>]
```

Comment:
Yup. Don't know if there's any syntax convention for default values though. Might just write it in text below.